### DISCUSSION TITLE
Incorrect Copy Button Behavior for Recipient Address on Coin Detail Page

### DISCUSSION THREAD

**Discussion by P-r-a-s-a-n-n-a** [spacescan-io/Spacescan-discussion#68] [Support] [answered]
noticed an issue on the coin detail page for the CAT token.When click on the "copy" button for the recipient address, it actually copies the sender address instead. This behavior is inconsistent with the expected functionality.

**Comment by nandhakumar2002** [comment, marked as answer]
fixed
